onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -146,7 +146,9 @@ def quantize_static(model_input,
                     nodes_to_exclude=[],
                     optimize_model=True,
                     use_external_data_format=False,
-                    calibrate_method=CalibrationMethod.MinMax):
+                    calibrate_method=CalibrationMethod.MinMax,
```

Comment:
We recently add an extra_options. It already has has  WeightSymmetric. Could you please align you code to that?
https://github.com/microsoft/onnxruntime/blob/2f2aaf2cf6151e75c4023fc2d0c9d044f5cd212c/onnxruntime/python/tools/quantization/quantize.py#L182 
https://github.com/microsoft/onnxruntime/blob/500f18badbb36e30d40f6dab9c4bf0fe277c60b8/onnxruntime/python/tools/quantization/onnx_quantizer.py#L45

Reply:
Thanks, I re-based and adopted the `extra_options` argument